study.md

```diff
@@ -44,21 +49,27 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+A view is essentially what is onscreen. A view state refers to interactions with
+aspects of the view. Such as a menu being pulled down, or other interactive
+functionality of the view itself.
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+Developing for the web as opposed to traditional apps have a number of advantages,
+one of the biggest being cross platform ability. This makes applications more readily
+available. I.e. you only need a browser so OS compatability are not as much of a
+a concern.
 ```
 
 ## List some technologies used to make SPA's
 
 What are some of the tools used to create single page applications?
 
 ```md
-<!-- your answer here -->
+AJAX is probably the biggest coupled with JSON. Ajax iteslf is a mutation of
```

Comment:
Don't forget front end frameworks like Ember, Angular, and React that are fantastic at handling view states and front end routing.